specification/cognitiveservices/Language.Conversations/common.tsp

```diff
@@ -17,28 +17,11 @@ using Azure.Core.Traits;
 namespace Azure.AI.Language.Conversations;
 
 @doc("Error Response")
-model ErrorResponse {
-    @doc("The error object.")
-    error: Error;
+model ErrorResponse is Azure.Core.Foundations.ErrorResponse{
 }
 
 @doc("The error object.")
-model Error extends Record<unknown> {
-    @doc("One of a server-defined set of error codes.")
-    code: ErrorCode;
-
-    @doc("A human-readable representation of the error.")
-    message: string;
-
-    @doc("The target of the error.")
-    target?: string;
-
-    @doc("An array of details about specific errors that led to this reported error.")
-    details?: Array<Error>;
-
-    @doc("An object containing more specific information than the current object about the error.")
-    innererror?: InnerErrorModel;
-}
+model Error is Azure.Core.Foundations.Error{}
```

Comment:
Shouldn't need to redeclare this or any other errors. `ErrorResponse` will pull all them in.